src/status_im/constants.cljs

```diff
@@ -112,5 +112,8 @@
 
 (def ^:const web3-send-transaction "eth_sendTransaction")
 (def ^:const web3-personal-sign "personal_sign")
+(def ^:const web3-get-logs "eth_getLogs")
+
+(def ^:const event-hash-transfer "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef")
```

Comment:
How is this calculated?

Reply:
keccak256 of Transfer event signature

Reply:
Can you add a comment? We might want to calculate this during compilation later for make the code more readable.